purchases/src/main/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -177,6 +179,7 @@ class Purchases internal constructor(
             log(LogIntent.WARNING, AUTO_SYNC_PURCHASES_DISABLED)
         }
 
+        offlineEntitlementsManager.updateProductEntitlementMappingsCacheIfStale()
```

Comment:
maybe we can just leave this line disabled and merge this into main? i.e.: if we don't actually _call_ it, it should be shippable, right?

Reply:
Yeah, that's possible. I will do that tomorrow.